src/status_im/ethereum/contracts.cljs

```diff
@@ -8,7 +8,14 @@
    :status/tribute-to-talk
    {:testnet "0xC61aa0287247a0398589a66fCD6146EC0F295432"}
    :status/stickers
-   {:testnet "0x39d16CdB56b5a6a89e1A397A13Fe48034694316E"}})
+   {:testnet "0x8cc272396be7583c65bee82cd7b743c69a87287d"
+    :mainnet ""}
+   :status/sticker-market
+   {:testnet "0x6CC7274aF9cE9572d22DFD8545Fb8c9C9Bcb48AD"
+    :mainnet ""}
+   :status/sticker-pack
```

Comment:
could please add descriptions for each stickers contract? why do we have 3 now instead 1 ?

Reply:
It's part of latest @3esmit changes. See https://notes.status.im/Q-sQmQbpTOOWCQcYiXtf5g#
AFAIK it allows easier upgrades.